pandas/tests/io/test_orc.py

```diff
@@ -0,0 +1,266 @@
+""" test orc compat """
+import datetime
+import distutils
+import os
+
+import numpy as np
+import pytest
+
+from pandas.compat import is_platform_windows
+
+import pandas as pd
+import pandas.util.testing as tm
+
+from pandas.io.orc import PyArrowImpl, get_engine, read_orc
+
+try:
+    import pyarrow  # noqa
+
+    if distutils.version.LooseVersion(pyarrow.__version__) < "0.13.0":
+        raise ImportError("pyarrow must be >= 0.13.0 for read_orc")
+
+    _HAVE_PYARROW = True
+except ImportError:
+    _HAVE_PYARROW = False
+
+pytestmark = pytest.mark.filterwarnings(
+    "ignore:RangeIndex.* is deprecated:DeprecationWarning"
+)
+
+
+@pytest.fixture
+def dirpath(datapath):
+    return datapath("io", "data", "orc")
+
+
+@pytest.fixture
+def pa():
+    if not _HAVE_PYARROW:
```

Comment:
If we just work with the assumption for now that we only have PyArrow to read we can really simplify this by making the skips global to the module and not having to use a fixture at all